test/expected/background_worker.out

```diff
@@ -0,0 +1,197 @@
+(1 row)
+
+COMMIT;
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  1
+(1 row)
+
+SELECT (wait_event IS DISTINCT FROM 'virtualxid') wait_event_changed
+FROM pg_stat_activity 
+WHERE application_name = 'Timescale BGW Scheduler Entrypoint' 
+AND datname = 'single_2';
+ wait_event_changed 
+--------------------
+ t
+(1 row)
+
+/*test stop*/
+SELECT _timescaledb_internal.stop_background_workers();
+ stop_background_workers 
+-------------------------
+ 
+(1 row)
+
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  0
+(1 row)
+
+/*make sure it doesn't break if we stop twice in a row*/
+SELECT _timescaledb_internal.stop_background_workers();
+ stop_background_workers 
+-------------------------
+ 
+(1 row)
+
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  0
+(1 row)
+
+/*test start*/
+SELECT _timescaledb_internal.start_background_workers();
+ start_background_workers 
+--------------------------
+ 
+(1 row)
+
+select pg_sleep(0.1);
+ pg_sleep 
+----------
+ 
+(1 row)
+
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  1
+(1 row)
+
+/*make sure start is idempotent*/
+SELECT backend_start as orig_backend_start 
+FROM pg_stat_activity 
+WHERE application_name = 'Timescale BGW Scheduler Entrypoint' 
+AND datname = 'single_2' \gset
+SELECT _timescaledb_internal.start_background_workers();
+ start_background_workers 
+--------------------------
+ 
+(1 row)
+
+select pg_sleep(0.1);
+ pg_sleep 
+----------
+ 
+(1 row)
+
+SELECT (backend_start = :'orig_backend_start'::timestamptz) backend_start_unchanged
+FROM pg_stat_activity 
+WHERE application_name = 'Timescale BGW Scheduler Entrypoint' 
+AND datname = 'single_2';
+ backend_start_unchanged 
+-------------------------
+ t
+(1 row)
+
+/*Make sure restart works from stopped worker state*/
+SELECT _timescaledb_internal.stop_background_workers();
+ stop_background_workers 
+-------------------------
+ 
+(1 row)
+
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  0
+(1 row)
+
+SELECT _timescaledb_internal.restart_background_workers();
+ restart_background_workers 
+----------------------------
+ 
+(1 row)
+
+select pg_sleep(0.1);
+ pg_sleep 
+----------
+ 
+(1 row)
+
+SELECT * FROM worker_counts;
+ launcher | single_scheduler | single_2_scheduler 
+----------+------------------+--------------------
+        1 |                0 |                  1
+(1 row)
+
```

Comment:
Create new database and extension in new database, make sure everything starts, drop database.